clang/test/CodeGenSYCL/free_function_int_header.cpp

```diff
@@ -0,0 +1,28 @@
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -triple spir64-unknown-unknown -sycl-std=2020 -fsycl-int-header=%t.h %s -emit-llvm -o %t.ll
+// RUN: FileCheck -input-file=%t.h %s
+// 
+// This test checks integration header contents for free functions with scalar,
+// pointer, simple struct parameters.
```

Comment:
Please update the comment. Simple struct parameters are not covered anymore

Reply:
Done.